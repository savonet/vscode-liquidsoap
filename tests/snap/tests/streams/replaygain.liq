#!../../liquidsoap ../test.liq

g = file.replaygain("file1.mp3")
print("Replay gain: #{g} dB")
if null.defined(g) and abs(null.get(g) - 7.37) < 0.05 then
  test.pass()
else
  test.fail()
end
