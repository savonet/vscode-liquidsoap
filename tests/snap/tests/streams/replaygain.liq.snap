>#!../../liquidsoap ../test.liq
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
>
>g = file.replaygain("file1.mp3")
#^^^^^^^^^^^^^^^^^^^^ source.liquidsoap
#                    ^ source.liquidsoap string.quoted.double.liquidsoap
#                     ^^^^^^^^^ source.liquidsoap string.quoted.double.liquidsoap
#                              ^ source.liquidsoap string.quoted.double.liquidsoap
#                               ^^ source.liquidsoap
>print("Replay gain: #{g} dB")
#^^^^^^ source.liquidsoap
#      ^ source.liquidsoap string.quoted.double.liquidsoap
#       ^^^^^^^^^^^^^^^^^^^^ source.liquidsoap string.quoted.double.liquidsoap
#                           ^ source.liquidsoap string.quoted.double.liquidsoap
#                            ^^ source.liquidsoap
>if null.defined(g) and abs(null.get(g) - 7.37) < 0.05 then
#^^ source.liquidsoap keyword.control.liquidsoap
#  ^^^^^^^^^^^^^^^^^ source.liquidsoap
#                   ^^^ source.liquidsoap keyword.operator.liquidsoap
#                      ^^^^^^^^^^^^^^^^^^^ source.liquidsoap
#                                         ^^^^ source.liquidsoap constant.numeric.float
#                                             ^^^^ source.liquidsoap
#                                                 ^^^^ source.liquidsoap constant.numeric.float
#                                                     ^ source.liquidsoap
#                                                      ^^^^ source.liquidsoap keyword.control.liquidsoap
>  test.pass()
#^^^^^^^^^^^^^^ source.liquidsoap
>else
#^^^^ source.liquidsoap keyword.control.liquidsoap
>  test.fail()
#^^^^^^^^^^^^^^ source.liquidsoap
>end
#^^^ source.liquidsoap keyword.control.liquidsoap
>